build-tools/automation/templates/core-build.yaml

```diff
@@ -6,10 +6,10 @@ steps:
   displayName: Prepare Solution
   inputs:
     projects: Java.Interop.sln
-    arguments: '-c $(Build.Configuration) -target:Prepare'
+    arguments: '-c $(Build.Configuration) -target:Prepare /v:diag'
```

Comment:
I would prefer not to add `diag`, as 99% of the time it just makes the logs harder to read.  We can look at recording a `binlog` and saving it as an artifact.